face.go

```diff
@@ -58,15 +60,25 @@ func NewRecognizer(modelDir string) (rec *Recognizer, err error) {
 	return
 }
 
-func (rec *Recognizer) recognize(imgData []byte, maxFaces int) (faces []Face, err error) {
+func (rec *Recognizer) Config(size int, padding float32, jittering int) {
```

Comment:
Would be better to pass config when creating recognizer, e.g. `NewRecognizerWithConfig`